Week-3/Session 7/Responsive images for a vacation website/egypt.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>Responsive Images Lab</title>
+   <link rel="stylesheet" href="styles.css">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+   <header>
+      <h1>Mediterranian Vacations</h1>
+   </header>
+   <main>
+      <h2>Egypt</h2>
+      
+      <!-- TODO: Add srcset attribute -->
+      <img srcset="egypt_600.jpg 1x,egypt_1200.jpg 2x,egypt_1800.jpg 3x" 
+            src="egypt_600.jpg" alt="Great Sphyinx and pyramid" class="stretch-image" >
+
+      <p>Egpyt is a land of enduring beauty and rich culture.
+         Explore the capital city of <strong>Cairo</strong>,
+         the <strong>Great Pyramids of Giza</strong>, and the
+         iconic <strong>Great Sphinx</strong>!
+      </p>
+
+      <!-- TODO: Add srcset and sizes attributes -->
+      <img srcset="luxor_600.jpg 600w,luxor_800.jpg 800w,luxor_1200.jpg 1200w" 
+         sizes="(min-width: 700px) 600px,
+         (min-width: 500px) 400px,
+         200px"
+         src="luxor_600.jpg" alt="Luxor Temple">
+
+      <p>
+         Packages include airfare, lodging, and several guided tours!
+      </p>
+      <ul>
+         <li>
+            <p>
+               <strong>Deluxe Package:</strong> 7 days in Cairo, Aswan, Luxor and Alexandria, 4 day Nile River cruise.
+            </p>
+         </li>
+         <li>
+            <p>
+               <strong>Super Package:</strong> 4 days in Cairo, 3 day Nile River cruise.
+            </p>
+         </li>
+         <li>
+            <p>
+               <strong>Express Package:</strong> 4 days in Cairo and upper Egypt on land.
+            </p>
+         </li>
+      </ul>
+
+      <!-- TODO: Add <picture> -->
```

Comment:
Is it done or still pending?

Reply:
It is done. Forgot to delete the comment.